py/specs/count.toml

```diff
@@ -0,0 +1,21 @@
+[setup]
+statement_files = ["sql/uservisits.sql"]
+statements = [
+    "copy uservisits from 's3://crate.amplab/data/tiny/uservisits/part-*' with (compression = 'gzip')",
+    "refresh table uservisits"
+]
+
+[[queries]]
+statement = "select count(*) from uservisits"
+iterations = 200
+
+[[queries]]
+statement = '''select count("cCode") from uservisits'''
+iterations = 200
+
+[[queries]]
+statement = '''select count("sourceIP") from uservisits'''
+iterations = 200
```

Comment:
I just wanted to add one more different count query with field instead of *. But I can delete one of the field query.